README.md

```diff
@@ -1,5 +1,28 @@
 # Open Operations Manifesto
 
+## I want to sign the manifesto. How can I do this?
+
+Thank you for intending to sign our Open Operations Manifesto.
+
+- As an organization, please add yourself to [`_data/organizations.yml`](https://github.com/SovereignCloudStack/open-operations-manifesto/blob/main/_data/organizations.yml) and upload your logo to [`assets`](https://github.com/SovereignCloudStack/open-operations-manifesto/tree/main/assets) by createing a pull request against `main`.
+- As an indivial signee, please add yourself to [`_data/signees.yml`](https://github.com/SovereignCloudStack/open-operations-manifesto/blob/main/_data/signees.yml) by createing a pull request against `main`.
+
+## I want to sign this manifesto but ain't confident with using GitHub
+
+Please reach out to <info@openoperations.org> and send us your confirmation (along with your logo). We're happy to help you!
```

Comment:
How is this email address managed?

Reply:
It's a list within our mailman3, which is distributed (at the moment) to Kurt, Felix and me.